src/elements/content-sidebar/skills/transcript/Transcript.js

```diff
@@ -55,7 +55,11 @@ class Transcript extends React.PureComponent<Props, State> {
      * @private
      * @return {void}
      */
-    UNSAFE_componentWillReceiveProps(): void {
+    componentDidUpdate(prevProps: Props): void {
+        if (prevProps === this.props) {
```

Comment:
This is already marked as a pure component. Is this needed?

Reply:
State is still changing so `componentDidUpdate` will still pass `shouldComponentUpdate` and an update will happen. We still only one this to run if the props are different.